src/main/scala/org/allenai/plugins/StylePlugin.scala

```diff
@@ -100,22 +102,20 @@ class SbtLogOutput[T <: FileSpec](logger: Logger) extends ScalastyleOutput[T] {
     case EndFile(file) => logger.verbose("end file " + file)
     case StyleError(file, clazz, key, level, args, line, column, customMessage) => {
       plevel(level)(location(file, line, column) + ": " +
-          ScalastyleOutput.findMessage(messageHelper, key, args, customMessage))
+        ScalastyleOutput.findMessage(messageHelper, key, args, customMessage))
     }
     case StyleException(file, clazz, message, stacktrace, line, column) =>
       logger.error(location(file, line, column) + ": " + message)
   }
 
-  private[this]
-  def plevel(level: Level)(msg: => String): Unit = level match {
+  private[this] def plevel(level: Level)(msg: => String): Unit = level match {
     case ErrorLevel => logger.error(msg)
     case WarningLevel => logger.warn(msg)
     case InfoLevel => logger.info(msg)
   }
 
-  private[this]
-  def location(file: T, line: Option[Int], column: Option[Int]): String =
+  private[this] def location(file: T, line: Option[Int], column: Option[Int]): String =
     (file.name +
-     line.map(n => ":" + n + column.map(":" + _).getOrElse(""))
-         .getOrElse(""))
+      line.map(n => ":" + n + column.map(":" + _).getOrElse(""))
+      .getOrElse(""))
```

Comment:
IIRC much of this plugin was copy-pasted from another sbt plugin's implementation.

I made this use a procedural `StringBuilder` because I am a REBEL.